src/main/java/com/minecolonies/core/compatibility/jei/JeiFakeLevel.java

```diff
@@ -0,0 +1,24 @@
+package com.minecolonies.core.compatibility.jei;
+
+import com.ldtteam.structurize.client.fakelevel.SingleBlockFakeLevel;
+import net.minecraft.client.Minecraft;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+@OnlyIn(Dist.CLIENT)

```

Comment:
why onlyin? dont think we should use that in 1.21

Reply:
Paranoia, mostly.  Since it's using client-only classes it must not be loaded in the server jar.  Though that should be the case even without this, it seems safer to make it explicit.